Editing corrections

bakerdb opened the pull request:
The new content re app protection was misplaced, so I moved it to precede step 1.

Also, the old step 1 (App asks for authorization) was still there, so I deleted it.

However, step 2 (EHR evaluates authorization request) seems to have disappeared.  I thought I had submitted changes for that section, but I don't see them in any of the patches.  Unless you have an explanation that I'm not seeing, I'll just copy it from the master, make the changes, and submit them.

jmandel commented:
I fixed the numbering and added back the "EHR evaluates authz request" step which looks like it got lost in a previous merge. Thanks!

bakerdb commented:
Thank you!

-Dixie


---- Original Message ----

> I fixed the numbering and added back the "EHR evaluates authz request" step which looks like it got lost in a previous merge. Thanks!
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/smart-on-fhir/smart-on-fhir.github.io/pull/32#issuecomment-85765047
